CAP/gap/DerivedMethods.gi

```diff
@@ -2489,6 +2531,42 @@ AddDerivationToCAP( DirectSumProjectionInPushout,
     
 end : Description := "DirectSumProjectionInPushout using the universal property of the direct sum" );
 
+##
+AddDerivationToCAP( EmbeddingOfEqualizer,
+                    [ [ KernelEmbedding, 1 ],
+                      [ JointPairwiseDifferencesOfMorphismsIntoDirectProduct, 1 ],
+                      [ PreCompose, 1 ],
+                      [ IsomorphismFromEqualizerToKernelOfJointPairwiseDifferencesOfMorphismsIntoDirectProduct, 1 ] ],
+                    
+  function( cat, A, diagram )
+    local kernel_of_pairwise_differences;
+    
+    kernel_of_pairwise_differences := KernelEmbedding( cat, JointPairwiseDifferencesOfMorphismsIntoDirectProduct( cat, A, diagram ) );
+    
+    return PreCompose( cat, IsomorphismFromEqualizerToKernelOfJointPairwiseDifferencesOfMorphismsIntoDirectProduct( cat, A, diagram ),
+                       kernel_of_pairwise_differences );
+    
+end : CategoryFilter := IsPreAbelianCategory,
```

Comment:
Why is `IsPreAbelianCategory` required?

Reply:
I think it's not. At the time I wrote the code, I thought it would be good style to add `CategoryFilter` and it survived my changes since then.